src/components/autocomplete/js/autocompleteController.js

```diff
@@ -359,8 +359,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
    * Handles input blur event, determines if the dropdown should hide.
    */
   function blur () {
-    hasFocus = false;
```

Comment:
Yes, this is so that we know when the element lost focus so that we can refocus when they leave the dropdown. This was related to ie 11.

Sent from my iPhone

> On Nov 4, 2015, at 3:43 PM, Topher Fangio <notifications@github.com> wrote:
> 
> In src/components/autocomplete/js/autocompleteController.js:
> 
> > @@ -359,8 +359,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
> >     * Handles input blur event, determines if the dropdown should hide.
> >     */
> >    function blur () {
> > -    hasFocus = false;
> @robertmesserle Can you think of any reason why the hasFocus was outside of the if statement? All tests pass after moving it and my manual testing didn't show any differences except that it fixed my bug.
> 
> —
> Reply to this email directly or view it on GitHub.
>